Frontend/ui-customer/src/Components/SearchLocationModel/View.purs

```diff
@@ -360,7 +360,7 @@ sourceDestinationEditTextView state push =
 searchResultsView :: forall w . SearchLocationModelState -> (Action  -> Effect Unit) -> PrestoDOM (Effect Unit) w
 searchResultsView state push =
   PrestoAnim.animationSet [
-    translateYAnimFromTop $ translateFullYAnimWithDurationConfig 550] $
+    translateYAnimFromTop $ translateFullYAnimWithDurationConfig 800] $
```

Comment:
This duration is for animation and not for debounce, Please update this in the debounce function